### PR TITLE
Created object first, then updated settings.

### PR DESCRIPTION
**Pull request by PwshAddict**
### What problem does this PR address?
CodeCoverage.xml fails to include all files from the codecoverage folder.
  
### Is there a related Issue?
Issue 38
  
### Have you...
- If you are willing, please enable me, as the Repo owner, to [edit your fork that contains your PR](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) so I can add any missing items such as the readme.md

### CODE REVIEW

File: Extension/PesterTask/PesterV10/Pester.ps1 [hunk 1]
```diff
@@ -110,30 +109,27 @@ if ($ExcludeTag) {
...
+    Select-object -ExpandProperty Fullname
 
     if ($Files) {
-        $CodeCoverage.Add('Path', $Files)
-        $CodeCoverage.Add('OutputPath', $CodeCoverageOutputFile)
+        $PesterConfig.CodeCoverage.Path = $Files
+        $PesterConfig.CodeCoverage.OutputPath = $CodeCoverageOutputFile
     }
-    else {
+    else {`
```
Comment (ChrisLGardner) [inline Extension/PesterTask/PesterV10/Pester.ps1]: Looks like a random backtick ended up here, can you remove it? It won't cause any harm being there with nothing after it but it's also not doing anything.

File: Extension/PesterTask/PesterV10/Pester.ps1 [hunk 2]
```diff
@@ -110,30 +109,27 @@ if ($ExcludeTag) {
...
-    $CodeCoverage['OutputFormat'] = "JaCoCo"
+    $PesterConfig.CodeCoverage.Enabled = $True
+    $PesterConfig.CodeCoverage.OutputFormat = "JaCoCo"
 
     if (-not $PSBoundParameters.ContainsKey('CodeCoverageFolder')) {
         $CodeCoverageFolder = $TestFolder
     }
     $Files = Get-ChildItem -Path $CodeCoverageFolder -include *.ps1, *.psm1 -Exclude *.Tests.ps1 -Recurse |
-        Select-object -ExpandProperty Fullname
+    Select-object -ExpandProperty Fullname
```
Comment (ChrisLGardner) [inline Extension/PesterTask/PesterV10/Pester.ps1]: Can you fix this indent back to where it was? I'm guessing VS Code did this automatically with it's formatter.

File: Extension/PesterTask/PesterV10/Pester.ps1 (hunk 1 above)
Comment (PwshAddict) [reply Extension/PesterTask/PesterV10/Pester.ps1]: Completed.

File: Extension/PesterTask/PesterV10/Pester.ps1 (hunk 2 above)
Comment (PwshAddict) [reply Extension/PesterTask/PesterV10/Pester.ps1]: Yes, it was a VSCode formatting issue.  This was fixed as well in my latest commit.

**Comment by ChrisLGardner** [comment]
Deployed as 10.1.23 and should now be publicly available. Thanks for this.

**Pull request merged**